src/app/node-data/component.ts

```diff
@@ -233,6 +233,10 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     this._nodeDataService.taints = this.taints;
   }
 
+  isDynamicKubletConfigSupported(): boolean {
```

Comment:
IMO, it doesn't matter but i prefer Getter to use field when simple `boolean` values needs to be considered. This comment is Opinionated can be ignored